core/trino-main/src/test/java/io/trino/block/BlockAssertions.java

```diff
@@ -596,4 +760,36 @@ public static RunLengthEncodedBlock createRLEBlock(long value, int positionCount
         BIGINT.writeLong(blockBuilder, value);
         return new RunLengthEncodedBlock(blockBuilder.build(), positionCount);
     }
+
+    private static Set<Integer> chooseNullPositions(int positionCount, float nullRate)
+    {
+        int nullCount = (int) (positionCount * nullRate);
+        if (nullCount == 0) {
```

Comment:
remove `if`. You can just do:
```
verify(nullRate != 0, "position count %s too small to have at least one null with rate %s", (Object) positionCount, nullRate);
```
No need to return anything here

Reply:
`nullRate` can be 0. In this case, we expect the return value to be empty.